myadmin-service-impls/student-services/src/main/java/za/ac/unisa/myadmin/student/services/models/StudentEntity.java

```diff
@@ -248,6 +248,11 @@ public StudentInfo toDto() {
 		info.setFirstNames(this.firstNames);
 		info.setInitials(this.initials);
 		info.setTitle(this.title);
+		if (this.birthDate != null) {
+			info.setDateOfBirth(this.birthDate);
```

Comment:
Small thing but when setting some date field equals to another date field I found it is better to a info.setDateOfBirth(new Date(this.birthDate)) because working with Objects you don't want to invertedly change the other object's value.